.github/workflows/tapioca-update.yml

```
@@ -0,0 +1,34 @@
+name: Tapioca Update
+on:
```

```suggestion
on:
  pull_request:
    paths:
      - .github/workflows/tapioca.yml
```

so that it's also run on this test.